### PR TITLE
IA case API migration to Kustomization framework

### PR DESCRIPTION
**Pull request by makowalski**
### JIRA link (if applicable) ###
[RIA-3807](https://tools.hmcts.net/jira/browse/RIA-3807)

### Change description ###
IA case API migration to Kustomization framework

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by jkumar-moj — DISMISSED** [review]

**Review by jkumar-moj — APPROVED** [review]

File: k8s/namespaces/ia/ia-case-api/demo.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+kind: HelmRelease
+metadata:
+  name: ia-case-api
+spec:
+  values:
+    java:
+      environment:
+        IA_FEEPAYMENT_ENABLED: "true"
+        IA_HOME_OFFICE_INTEGRATION_ENABLED: "true"
+        EM_BUNDLER_URL: "http://em-ccdorc-demo.service.core-compute-demo.internal"
```
Comment (timja) [inline k8s/namespaces/ia/ia-case-api/demo.yaml:11]: is there a reason this is here? and not in the helm chart? (in progress maybe?)

File: k8s/namespaces/ia/ia-case-api/ia-case-api.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  name: ia-case-api
+spec:
+  releaseName: ia-case-api
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/ia-case-api
+  values:
+    java:
+      replicas: 2
```
Comment (timja) [inline k8s/namespaces/ia/ia-case-api/ia-case-api.yaml:13]: is there a reason auto scaling isn't enabled? recommended but not required and could be a separate PR

```suggestion
      replicas: 2
      autoscaling:
        enabled: true
        maxReplicas: 5
        targetCPUUtilizationPercentage: 80
```

**Review by timja — APPROVED** [review]
2 minor questions but approved

File: k8s/namespaces/ia/ia-case-api/ia-case-api.yaml (hunk 2 above)
Comment (makowalski) [reply k8s/namespaces/ia/ia-case-api/ia-case-api.yaml:13]: we don't have it in other apps too, let me create separate PR with autoscaling property

File: k8s/namespaces/ia/ia-case-api/demo.yaml (hunk 1 above)
Comment (makowalski) [reply k8s/namespaces/ia/ia-case-api/demo.yaml:11]: This feature was moved to documents-api, let me clean up with follow up PR
